perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -367,18 +368,19 @@ sub map : Private {
 
 =head2 check_has_permission_to
 
-Ensure the currently logged-in user has a particular permission that applies
+Ensure the currently logged-in user has particular permission(s) that applies
```

Comment:
This wording isn't clear if any or all of the given permissions are required (though the code is clear).